Gruntfile.js

```diff
@@ -15,8 +15,7 @@ module.exports = function (grunt) {
     configFile: __dirname + '/src/server/config/kibana.yml',
 
     nodeVersion: '0.10.35',
-    platforms: ['darwin-x64', 'linux-x64', 'linux-x86', 'windows'],
-    services: [ [ 'launchd', '10.9'], [ 'upstart', '1.5'], [ 'systemd', 'default'], [ 'sysv', 'lsb-3.1' ] ],
+    platforms: ['linux-x64'],
```

Comment:
@alexweltman i can look into the parameters. what was happening was even though it wasn't building the options it required the installation of extra ruby gems, so i just removed the references to those gems.

i removed the services because we don't need that step at all. for the platforms i can look at the build options